Utils/Dataflow/017_adjustMetadata/adjustMetadata.py

```diff
@@ -154,9 +154,14 @@ def input_events(data):
 
 
 def transform_chain_data(data):
-    """ Transform chain_data into array of integers and get chain_id from it.
+    """ Transform chain_data into proper task metadata fields.
 
-    chain_id is the taskid of the task chain's root.
+    New/updated fields:
+      - chain_data:          task IDs of all previous tasks in chain
```

Comment:
Sorry, got too busy with actual problems with logic and forgot to bark at trivial things: `chain_data` includes the current task. I'm not sure that this is the right place for explaining what exactly a task chain is, we have READMEs and 009 for that, so I think dropping `previous` would be enough.

Reply:
Fixed: 6df3e94
I couldn't say "all tasks", for "all" may include tasks _after_ the current one -- so it would be not less confusing.